README.md

```diff
@@ -110,4 +110,19 @@ fragment.init();
 //  <p>p1</p>,
 //  <p>p2</p>
 // ]
+```
+
+## oDom
+oDom allows to get element from DOM using query selector (css selectors). Returns ojs object.
+Returns null in all other cases (element not found or wrong arguments);
+```ts
+oDom(selector: String, parentNode: DOMElement|ojsInstance = document)
+```
+#### Example:
+```js
+import o,{ oDom } from 'ojs-core';
+
+const pElement = oDom('#someParagraph');
+
+pElement.text('new paragraph text set using ojs API');
```

Comment:
Spoko by było rozwinąć ten przykład. Bo nie jest do końca jasne jak to działa z drugim argumentem. Nie czytałem jeszcze implementacji i nie rozumiem do końca. W sensie to parentNode to jest Node w którym ma szukać jeśli podam?

Reply:
W sumie jak spojrzałem w implementacje to tutaj też widać, że defaultowo to document. Ale nie skapnąłem się od razu, to może załóżmy, że inni też będą tacy głupi jak ja i lepiej to jakoś bardziej podkreślić